gears/flywheel/mriqc.json

```diff
@@ -12,12 +12,22 @@
         "anatomy_t1w",
         "anatomy_t2w"
       ]
+    },
+    "save_derivatives": {
+      "default": false,
+      "description": "Save the derivative JSON file generated by MRIQC algorithm.",
+      "type": "boolean"
+    },
+    "save_outputs": {
+      "default": false,
+      "description": "Save all outputs generated by MRIQC algorithm. These results will be saved in one single zip file.",
+      "type": "boolean"
     }
   },
   "custom": {
-    "docker-image": "flywheel/mriqc:v0.3.2"
+    "docker-image": "flywheel/mriqc:v0.3.3"
   },
-  "description": "MRIQC extracts no-reference IQMs (image quality metrics) from structural (T1w and T2w) and functional MRI (magnetic resonance imaging) data. Note, this gear only supports the generation of individual scan reports; group reports are not generated.",
+  "description": "MRIQC extracts no-reference IQMs (image quality metrics) from structural (T1w and T2w) and functional MRI (magnetic resonance imaging) data. Note, this gear only supports the generation of individual scan reports; group reports are not generated. Also note, for the auto-detection config option to work for this gear, the follow gears must be run beforehand: (1) dicom-mr-classifier then (2) dcm2niix (version > v0.3.1).",
```

Comment:
(2) dcm2niix (version 0.3.1 or higher )

Reply:
Thanks for this feedback. This is fixed now.